File: Ruby/Introduction to Ruby/Intro to Programming/2. Variables/age.rb
puts "How old are you?"
age = gets.chomp.to_i
def get_future_age (current_age, years_added)
    "In #{years_added} years you will be #{age + years_added}"
end
puts get_future_age(age, 10)
puts get_future_age(age, 20)
puts get_future_age(age, 30)
puts get_future_age(age, 40)

File: Python/PY100/intro-to-python/3-variables/age.py
AGE = 10
print("You are " + str(AGE) + " years old!")


def age_calculator(years):
    return "In " + str(years) + " years you'll be " + str(AGE + years)


print(age_calculator(10))
print(age_calculator(20))
print(age_calculator(30))
print(age_calculator(40))
